spec/lib/hamster/associable/associable_spec.rb

```diff
@@ -1,7 +1,7 @@
 require "hamster/hash"
 require "hamster/vector"
 
-describe Hamster::Associable do
```

Comment:
What is the benefit of using the `Rspec.describe` syntax rather than `describe`?

Reply:
they're semantically equivalent, but `Rspec.describe` is "nicer" because it doesn't require monkey patching the global namespace. It's now the Rspec recommended way to do this.

Reply:
I'm wondering what the problem with defining methods on `main` and its ancestors (i.e. the "global namespace") is. Of course, you could have a naming clash if another library defined a method called `describe`, but I've never seen another one which does.

RSpec's DSL is the best part about it. This change makes the DSL uglier and more verbose. However, it is good that there is the *option* of namespacing, for projects which really do have a different method called `describe` in the "global namespace".

Reply:
Mostly a principle thing, though in practice we've also found it seems less "magical" to people. Doesn't particularly matter here.